### ISSUE TITLE
Implement middleware Multer

### ISSUE THREAD

**Issue by rsaz** [expressots/expressots#106]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

We have a [middleware service](https://github.com/expressots/expressots/blob/main/packages/core/src/middleware/middleware-service.ts) where we do add services that can be used dynamically by the developer during the application development like so:

```typescript
@provide(App)
export class App extends AppExpress {
    private middleware: IMiddleware;
    private provider: IProvider;

    constructor() {
        super();
        this.middleware = container.get<IMiddleware>(Middleware);
        this.provider = container.get<IProvider>(Provider);
    }

    protected configureServices(): void {
        this.middleware.addBodyParser();
        this.middleware.setErrorHandler();
    }

    protected postServerInitialization(): void {
        if (this.isDevelopment()) {
            this.provider.envValidator.checkAll();
        }
    }

    protected serverShutdown(): void {}
}
```

The middleware are added in the middleware service as mentioned above as well as it's interfaces implemented [here](https://github.com/expressots/expressots/tree/main/packages/core/src/middleware/interfaces).
The interfaces are the options that can be optionally passed to the middleware.

This task has the objective of implementing the middleware [Multer](https://www.npmjs.com/package/multer) embedded in the ExpressoTS platform as the other middlewares.

More information here: https://expressjs.com/en/resources/middleware/multer.html

### Steps to reproduce

N/A

### Expected behavior

Multer implemented as an option in middleware service

### Package version

2.2.1

### Node.js version

> 18

### In which operating systems have you tested?

- [X] macOS
- [X] Windows
- [X] Linux

**Comment by shravan20** [comment]
@rsaz: Can you assign this to me? I would like to pick this up. Are there any deadlines to its completion or release?

**Comment by rsaz** [comment]
> @rsaz: Can you assign this to me? I would like to pick this up. Are there any deadlines to its completion or release?

Assigned. Looking forward to hear from you regarding this issue.

**Comment by ArslanYM** [comment]
Is this issue open?

**Comment by rsaz** [comment]
> Is this issue open?

Still open. I can assign to you. Let me know!

**Comment by ArslanYM** [comment]
Yes, id like to try my hand at this. Please assign me

**Comment by rsaz** [comment]
Closed #120
